integration-test/src/test/java/org/sagebionetworks/ITDownloadStorageReportTest.java

```diff
@@ -52,32 +41,18 @@ public static void afterClass() throws Exception {
 	}
 
 	@Test
-	public void generateReportAndGet() throws SynapseException, InterruptedException {
+	public void generateReportUnauthorized() throws SynapseException, InterruptedException {
 		String jobToken = synapse.generateStorageReportAsyncStart(StorageReportType.ALL_PROJECTS);
-		DownloadStorageReportResponse response =  null;
-		boolean csvCreated = false;
-		while (!csvCreated) {
+		boolean jobProcessed = false;
+		while (!jobProcessed) {
 			Thread.sleep(RETRY_TIME);
 			try {
-				response = synapse.generateStorageReportAsyncGet(jobToken);
-				if (response.getResultsFileHandleId() != null) {
-					csvCreated = true;
-				}
-			} catch (SynapseResultNotReadyException e) {
-				assertNotEquals(e.getJobStatus().getJobState(), AsynchJobState.FAILED);
+				synapse.generateStorageReportAsyncGet(jobToken);
+				fail("Expected exception");
+			} catch (SynapseForbiddenException e) {
```

Comment:
A forbidden exception tells us the call reached the manager, which is sufficient for testing the controller